ui/src/kapacitor/components/config/SMTPConfig.tsx

```diff
@@ -54,7 +71,7 @@ class SMTPConfig extends Component {
             id="smtp-port"
             type="text"
             ref={r => (this.port = r)}
-            defaultValue={port || ''}
+            defaultValue={port.toString() || ''}
```

Comment:
if this is already enforced as a string type, why the need to convert it?

Reply:
Port is a number but defaultValue requires a string